Microsoft.DotNet.Interactive.Jupyter/KernelExtensions.cs

```diff
@@ -35,6 +36,18 @@ public static T UseDefaultMagicCommands<T>(this T kernel)
             return kernel;
         }
 
+        public static CSharpKernel UseJupyterHelpers(
+            this CSharpKernel kernel)
+        {
+            var command = new SubmitCode($@"
```

Comment:
this would be safer with 
```
            kernel.DeferCommand(
                new SubmitCode($@"
#r ""{typeof(Kernel).Assembly.Location.Replace("\\", "/")}""
using static {typeof(Kernel).FullName};
"));
```